packages/netlify-cms-core/src/components/Collection/Entries/EntryCard.js

```diff
@@ -1,12 +1,25 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
```

Comment:
We can't use hooks until the next major release (😞) - multiple React instances will crash the app, which occurs often in React SSG's like Gatsby. Worth discussing whether we want to break the rules on this one, haven't reviewed the stance in a while.

Reply:
Switched to class component (created an `Asset` component in `netlify-cms-ui-default`).
As for the hook in the editor image component, had to change `toPreview` to an async method which meant a bunch of other changes to support that in the markdown widget.
The markdown widget uses `renderToString` which only renders using the initial state, so had to change `markdownToHtml` to async as well and change `MarkdownPreview` to a class component.